Model/LdapUser.php

```diff
@@ -0,0 +1,260 @@
+<?php
+
+namespace FR3D\LdapBundle\Model;
+
+use Symfony\Component\Security\Core\User\AdvancedUserInterface;
+
+/**
+ * Simple user object.
+ */
+class LdapUser implements LdapUserInterface, AdvancedUserInterface
```

Comment:
This can be simplified even more replacing `AdvancedUserInterface` with `UserInterface`

Since this class is not persisted there is no way to control locked, expired, enabled attributes.

However an empty `setEnabled()` method must be left due #2